This patch fixes errors on powerpc like below

| lnx_video.c: In function 'unmapVidMem':
| lnx_video.c:472:24: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]

Its in 1.12 branch 

http://cgit.freedesktop.org/xorg/xserver/commit/?h=server-1.12-branch&id=eb3377ffb8a7baa26c9831e56ed782d48b28fa71

Upstream-Status: Backport

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: xorg-server-1.11.2/hw/xfree86/os-support/linux/lnx_video.c
===================================================================
--- xorg-server-1.11.2.orig/hw/xfree86/os-support/linux/lnx_video.c	2012-05-01 06:32:34.186489019 -0700
+++ xorg-server-1.11.2/hw/xfree86/os-support/linux/lnx_video.c	2012-05-01 06:49:14.926537452 -0700
@@ -469,11 +469,11 @@
 static void
 unmapVidMem(int ScreenNum, pointer Base, unsigned long Size)
 {
-    memType alignOff = (memType)Base 
-	- ((memType)Base & ~(getpagesize() - 1));
+    uintptr_t alignOff = (uintptr_t)Base 
+	- ((uintptr_t)Base & ~(getpagesize() - 1));
 
-    DebugF("alignment offset: %lx\n",alignOff);
-    munmap((caddr_t)((memType)Base - alignOff), (Size + alignOff));
+    DebugF("alignment offset: %lx\n",(unsigned long)alignOff);
+    munmap((void*)((uintptr_t)Base - alignOff), (Size + alignOff));
 }
 
 
